project/Release.scala

```diff
@@ -0,0 +1,58 @@
+import sbt._
+import Keys._
+import bintray.BintrayKeys._
+import sbtrelease.ReleasePlugin.autoImport._
+import sbtrelease.ReleaseStateTransformations._
+
+object Release {
+  def settings = releaseSettings ++ publishEnabled
+
+  def releaseSettings = Seq(
+    crossSbtVersions := Vector("1.3.10", "0.13.16"),
+    publishMavenStyle := false,
+    bintrayRepository := "sbt-plugins",
+    bintrayPackage := name.value,
+    bintrayOrganization := Some("allenai"),
+    releaseProcess := releaseProcessSteps,
+    releaseUseGlobalVersion := false,
+    organization := "org.allenai.plugins",
+    organizationHomepage := Some(url("https://www.allenai.org")),
+    licenses := Seq("Apache-2.0" -> url("https://www.apache.org/licenses/LICENSE-2.0.html")),
+    homepage := Some(url(s"https://github.com/allenai/sbt-plugins")),
+    scmInfo := Some(
+      ScmInfo(
+        url("https://github.com/allenai/sbt-plugins"),
+        "git@github.com:allenai/sbt-plugins.git"
+      )
+    )
+  )
+
+  def releaseProcessSteps = Seq[ReleaseStep](
+    checkSnapshotDependencies,
+    inquireVersions,
+    runClean,
+    releaseStepCommandAndRemaining("^test"),
+    releaseStepCommandAndRemaining("^ scripted"),
+    setReleaseVersion,
+    commitReleaseVersion,
+    tagRelease,
+    releaseStepCommandAndRemaining("^publish"),
```

Comment:
These 3 `releaseStepCommandAndRemaining` changes with the `^` are the main changes. Where `+` tells sbt to run the command with all versions of scala specified in the `crossScalaVersions` key -- for example, `+test` will run `test` on all scala versions -- `^` tells sbt to run the command with all version of sbt specified in the `crossSbtVersions` key.